src/System.Memory/tests/ReadOnlySpan/IndexOfAny.byte.cs

```diff
@@ -61,12 +61,14 @@ public static void ZeroLengthIndexOfTwo_Byte()
             ReadOnlySpan<byte> sp = new ReadOnlySpan<byte>(Array.Empty<byte>());
             int idx = sp.IndexOfAny<byte>(0, 0);
             Assert.Equal(-1, idx);
+            Assert.Equal(idx, sp.IndexOfAny(new byte[2]));
         }
 
         [Fact]
         public static void DefaultFilledIndexOfTwo_Byte()
         {
             Random rnd = new Random(42);
+            Span<byte> values = new Span<byte>(new byte[2]);
```

Comment:
Do we need to store this into a local, or could we make the test cleaner and easier to understand at the IndexOfAny call site by removing this and changing the below to something like just:
```C#
Assert.Equal(0, span.IndexOfAny(target0, target1));
Assert.Equal(0, span.IndexOfAny(new[] { target0, target1 }.AsSpan()));
```
?